tests/test_modeling_tf_camembert.py

```diff
@@ -0,0 +1,50 @@
+# coding=utf-8
+# Copyright 2018 The Google AI Language Team Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from transformers import is_tf_available
+
+from .utils import require_tf, slow
+
+
+if is_tf_available():
+    import tensorflow as tf
+    import numpy as np
+    from transformers import TFCamembertModel
+
+
+@require_tf
+class TFCamembertModelIntegrationTest(unittest.TestCase):
+    @slow
+    def test_output_embeds_base_model(self):
+        model = TFCamembertModel.from_pretrained("jplu/tf-camembert-base")
```

Comment:
Use the model identifier `"jplu/tf-camembert-base"` at the moment because in 
https://github.com/huggingface/transformers/blob/cf72479bf11bf7fbc499a518896dfd3cafdd0b21/src/transformers/modeling_tf_camembert.py#L33
there are no official tf model names defined. Eventually, add `'camembert-base: "https://s3.amazonaws.com/models.huggingface.co/bert/camembert-base-tf_model.h5"` there? @jplu @thomwolf

Reply:
Let's see what @julien-c think about that as well

Reply:
I think it's fine to use `"jplu/tf-camembert-base"` in the integration test, as it's the official TF camembert model and gives some visibility/credit to @jplu :)

Reply:
Ok great. Good to merge then for me!